packages/hint-compat-api/src/_locales/en/messages.json

```
@@ -22,5 +22,13 @@
     "html_name": {
         "description": "html hint metadata name",
         "message": "Supported HTML features"
+    },
+    "learnMoreCSS": {
+        "description": "Text to show for the documentation link",
+        "message": "Learn more about the CSS property"
```

May need to tweak this text a bit. These could be properties, values, selectors, etc. Plus we should probably call out MDN somewhere in the link text. How about this?
```suggestion
        "message": "Learn more about this CSS feature on MDN"
```